src/azul/hmac.py

```diff
@@ -0,0 +1,28 @@
+import chalice
+import requests
+from requests_http_signature import HTTPSignatureAuth
+
+from azul import config
+
+
+def validate(current_request):
```

Comment:
```suggestion
def verify(current_request):
```

The term *verify* seems to be more commonly used in conjunction with HMAC. For example: https://en.wikipedia.org/wiki/HMAC